README.md

```diff
@@ -159,6 +159,21 @@ plan:
         Authorization: Basic aHR0cHdhdGNoOmY=
         X-Foo: Bar
         X-Bar: Bar {{ memory.headers.token }}
+
+  - name: Same request multiple times
+    request:
+      url: /counter
+    times: 10
+
+  - name: One request with random item
+    request:
+      url: /api/users/{{ item }}
+    with_items:
+      - 70
+      - 73
+      - 75
+    shuffle: true
+    times: 1
```

Comment:
For me, the hardest thing to understand about this new field is if it means to perform this block N times or N requests. I think it has to be clarified somehow.

Reply:
Right, I was struggling with the naming too.  
Current implementation does N requests, so maybe just name it `requests` or `amount_of_requests`?

Reply:
It is hard to find a right name, here. 😅 Also, for values greater than the amount of items is a bit weird. Other names that come to my mind are `repeat`, `subset`, `pick`, but not really convinced

Reply:
I kinda like `repeat`, but it has the same problem that it's hard to distinguish between block and requests.  
So I think we need to choose something more specific like `repeat_request`.

But then again it might be misleading. Does `repeat: 1` means 1 or 2 requests are sent? :smile:  
Just decide on what you like more, I would be fine with all of them.

Reply:
I have one idea. Let me know if it make sense:

I start liking more `pick`. This field must be less or equal to the number of items in the `with_items` field. And for simple request blocks, it doesn't make sense, so we should remove it, because it is where all the misleading part starts.

Reply:
Alright, done.
Made the code a lot easier too.

Reply:
Wow. Indeed. I think that now it is much easier to understand. 💯💯